pom.xml

```diff
@@ -211,13 +162,116 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>migrate-db</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
```

Comment:
Ernie is going to tweak this to look more like the profile in exam (linked above).  I took his "That was my assumption" to be referring to my comment that things built properly with all of it stuffed within `db-migrate`.